### PR TITLE
Resolve projectRef for the FM direct controller

### PR DESCRIPTION
**Pull request by ziyue-101**
### Change description

Resolve project Ref for hub controller
### Tests you have done

- [ ] Run `make ready-pr` to ensure this PR is ready for review.
- [ ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

File: pkg/controller/direct/utils/projectref.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package logging
+package utils
```
Comment (acpana) [inline pkg/controller/direct/utils/projectref.go:15]: this may be a bit if bikeshedding but could we rename this pkg to `references` ?

**Review by acpana — COMMENTED** [review]
/hold

one bikeshed-y comment, can be overridden.

/lgtm
/approve

File: pkg/controller/direct/utils/projectref.go (hunk 1 above)
Comment (ziyue-101) [reply pkg/controller/direct/utils/projectref.go]: Done!

Comment (yuwenma) [reply pkg/controller/direct/utils/projectref.go]: I share the same bikeshedding as @acpana 🤣

**Comment by yuwenma** [comment]
/lgtm
/approve

Thanks.

**Pull request merged**